oleh-burger-app/src/components/CreatedOrders/CreatedOrders.js

```diff
@@ -0,0 +1,125 @@
+import './CreatedOrders.css'
+
+import SingleOrder from './SingleOrder/SingleOrder'
+import Loader from '../Main/Loader/Loader'
+
+import {useState, useEffect} from 'react';
+import axios from "axios";
+
+const CreatedOrders = () => {
+    const loadOrders = async () => await axios.get("https://burger-api-xcwp.onrender.com/orders");
+
+    const [orders, setOrders] = useState([]);
+    const [ordersPortion, setOrdersPortion] = useState(undefined);
+    const [pageQuantity, setPageQuantity] = useState(0);
+    const [prevBtnActive, setPrevBtnActive] = useState('disabled');
+    const [nextBtnActive, setNextBtnActive] = useState('');
+    const [curentPage, setCurentPage] = useState(0);
```

Comment:
`useState `hooks always at the top

```suggestion
    const [orders, setOrders] = useState([]);
    const [ordersPortion, setOrdersPortion] = useState(undefined);
    const [pageQuantity, setPageQuantity] = useState(0);
    const [prevBtnActive, setPrevBtnActive] = useState('disabled');
    const [nextBtnActive, setNextBtnActive] = useState('');
    const [curentPage, setCurentPage] = useState(0);
    
    const loadOrders = async () => await axios.get("https://burger-api-xcwp.onrender.com/orders");
```